set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
+
+if (save) {
+  var fileName = path.join('..', '..', 'package.json')
```

Comment:
Wouldn't cwd be better? That way this doesn't need to be updated when files change location.

as suggested by @gr2m in https://github.com/hoodiehq/camp/issues/2#issuecomment-249711353